### PR TITLE
numeral systems transferring

### PR DESCRIPTION
**Pull request by romos21**
added numeral systems transferring task

### CODE REVIEW

File: binnary.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="assets/styles/binnary.css"/>
+    <title>Title</title>
+</head>
+<body>
+<input placeholder="input value" id="value"/>
```
Comment (ghost) [inline binnary.html]: _UI_: add tips/placeholder so the user can understand how the input data should look like. I can't understand at the first time how my input string should look like

File: binnary.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<br/>
+<input placeholder="input current numeral system" id="numSystemValue"/>
```
Comment (ghost) [inline binnary.html]: _UI_: add tips/placeholder so the user can understand how the input data should look like

File: assets/js/binnary.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+let curValue='';
+let curNumSystemValue='';
+let curResultNumSystemValue='';
+
+numSystemValue.onchange=()=>curNumSystemValue=parseInt(numSystemValue.value);
+value.onchange=()=>curValue=parseInt(value.value);
+resultNumSystemValue.onchange=()=>curResultNumSystemValue=parseInt(resultNumSystemValue.value);
+
+function NumSystemChange(value,curNumSystem,resultNumSystem){
+    if(curNumSystem!==10){
```
Comment (ghost) [inline assets/js/binnary.js:16]: The program doesn't convert from binary to the decimal system.

**Review by ghost — COMMENTED** [review]
@romos21 consider adding these fixes

File: assets/js/binnary.js (hunk 3 above)
Comment (romos21) [reply assets/js/binnary.js:16]: fixed

File: binnary.html (hunk 2 above)
Comment (romos21) [reply binnary.html]: fixed

File: binnary.html (hunk 1 above)
Comment (romos21) [reply binnary.html]: fixed

**Comment by ghost** [comment]
Great! Merging it

**Pull request merged**